PayItForward.Models/UserDTO.cs

```diff
@@ -1,13 +1,21 @@
-﻿namespace PayItForward.Models
+﻿using System.Collections.Generic;
+
+namespace PayItForward.Models
 {
     public class UserDTO
     {
```

Comment:
Do you have reason to make these virtual

Reply:
Yes, in order to make UnitTests

Reply:
What problems did you have when not making them virtual?